pom.xml

```diff
@@ -126,191 +137,393 @@
+            </sortPom>
+          </pom>
+          <upToDateChecking>
+            <enabled>true</enabled>
+          </upToDateChecking>
+        </configuration>
+        <executions>
+          <execution>
+            <id>spotless-check</id>
+            <goals>
+              <goal>check</goal>
+            </goals>
+            <phase>validate</phase>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <groupId>com.mycila</groupId>
+        <artifactId>license-maven-plugin</artifactId>
+        <version>${maven.license.plugin.version}</version>
+        <configuration>
+          <licenseSets>
+            <licenseSet>
+              <header>${license.header.resource}</header>
+              <includes>
+                <include>**/*.java</include>
+              </includes>
+            </licenseSet>
+          </licenseSets>
+          <failIfMissing>true</failIfMissing>
+          <aggregate>true</aggregate>
+          <strictCheck>true</strictCheck>
+          <mapping>
+            <java>DOUBLESLASH_STYLE</java>
+          </mapping>
+        </configuration>
+        <dependencies>
+          <dependency>
+            <groupId>org.eclipse.jetty.toolchain</groupId>
+            <artifactId>jetty-header-templates</artifactId>
+            <version>1.0</version>
+          </dependency>
+        </dependencies>
+        <executions>
+          <execution>
+            <id>check-headers</id>
+            <goals>
+              <goal>check</goal>
+            </goals>
+            <phase>validate</phase>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-enforcer-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>enforce-build-env</id>
+            <goals>
+              <goal>enforce</goal>
+            </goals>
+            <phase>validate</phase>
+            <configuration>
+              <rules>
+                <requireMavenVersion>
+                  <version>[${maven.version.minimum},)</version>
+                  <message>[ERROR] OLD MAVEN [${maven.version.minimum}] in use, ${project.artifactId} ${project.version} requires Maven ${maven.version.minimum} or newer</message>
+                </requireMavenVersion>
+                <requireJavaVersion>
+                  <version>[${jdk.version.minimum},)</version>
+                  <message>[ERROR] OLD JDK [${java.version}] in use. ${project.artifactId} ${project.version}  requires JDK ${jdk.version.minimum} or newer</message>
+                </requireJavaVersion>
+                <requireUpperBoundDeps />
+              </rules>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-remote-resources-plugin</artifactId>
```

Comment:
removed but I'm curious to see github says "must" have license file in github repository.
eclipse probably, I don't know. If yes I'm interested to see the link.
At Apache the only place where it's mandatory is the sources release distribution (not mandatory in the sources repo)

Reply:
If you want the LICENSE section on the github repo metadata to be accurate, you must have the license present in the root of the repo.
Otherwise it's "discovered" (and badly).

Reply:
right. it's only a matter of Github UI. so nothing related to any orgs.